From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: "Jan Alexander Steffens (heftig)" <heftig@archlinux.org>
Date: Fri, 30 May 2025 00:12:36 +0200
Subject: [PATCH] Fix compilation with GCC 15

For webrtc-audio-processing v1.3. Also see
e9c78dc4712fa6362b0c839ad57b6b46dce1ba83 upstream.
---
 webrtc/api/task_queue/task_queue_base.h | 1 +
 webrtc/rtc_base/trace_event.h           | 1 +
 2 files changed, 2 insertions(+)

diff --git a/webrtc/api/task_queue/task_queue_base.h b/webrtc/api/task_queue/task_queue_base.h
index 90b1efd31ebe..7403ee971e37 100644
--- a/webrtc/api/task_queue/task_queue_base.h
+++ b/webrtc/api/task_queue/task_queue_base.h
@@ -11,6 +11,7 @@
 #define API_TASK_QUEUE_TASK_QUEUE_BASE_H_
 
 #include <memory>
+#include <cstdint>
 
 #include "api/task_queue/queued_task.h"
 #include "rtc_base/system/rtc_export.h"
diff --git a/webrtc/rtc_base/trace_event.h b/webrtc/rtc_base/trace_event.h
index a0b788fbf943..db37557267f0 100644
--- a/webrtc/rtc_base/trace_event.h
+++ b/webrtc/rtc_base/trace_event.h
@@ -7,6 +7,7 @@
 #define RTC_BASE_TRACE_EVENT_H_
 
 #include <string>
+#include <cstdint>
 
 #include "rtc_base/event_tracer.h"
 
